cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -380,6 +388,9 @@ def __init__(self, *args, **kwargs):
         self._remote_api = remote_api.VmdkDriverRemoteApi()
         self._storage_profiles = []
         self._volume_type_by_backend = None
+        self._kmip_api = kmip.KMIPRestApiClient(
```

Comment:
what happens here if the config setting isn't set, the default is used and it tries to connect to it at startup?  Shouldn't we set the default to None and not try and create a client object here if the defaults are set?